workspaces/arborist/lib/override-set.js

```diff
@@ -193,10 +193,15 @@ class OverrideSet {
         return first
       }
     }
+
+    // The override sets are incomparable. Neither one contains the other.
     log.silly('Conflicting override sets', first, second)
   }
   
   static doOverrideSetsConflict (first, second) {
+    // If override sets contain one another then we can try to use the more specific one.
+    // This is imperfect, since we may lose some of the context of the less specific override
+    // set. But the more specific override set is more important to propagate, so we go with it.
```

Comment:
☝️ Does this comment belong here? The helper is `doOverridesSetsConflict` but the comment is about choosing the more specific override. It's unclear how the conflict bit and the picking of more specific is done in this bit.